src/pages/DataAccessRequestApplication.js

```diff
@@ -428,12 +467,48 @@ class DataAccessRequestApplication extends Component {
   };
 
   verifyStep2() {
+    //defined attribute keys for dynamic DUL based questions
+    const dulInvalidCheck = () => {
+      const activeQuestions = this.state.formData.activeDULQuestions;
+
+      const dulQuestionMap = {
+        'geneticStudiesOnly': 'gsoAcknowledgement',
+        'publicationResults': 'pubAcknowledgement',
+        'diseaseRestrictions': 'dsAcknowledgement',
+        'ethicsApprovalRequired': 'irbDocument',
+        'collaboratorRequired': 'collaborationLetter'
+      };
+      let result = false;
+
+      //NOTE: verify validation runs correctly here
```

Comment:
This could use some extra documentation. I'm not following all of the validation rules here.

Reply:
It was more of a note for me during development, but a small explanation for this block is still a good idea